stats/stats_test.go

```diff
@@ -724,3 +724,17 @@ func TestNaN(t *testing.T) {
 // should get 3 buckets 0-1 with count 1
 // 1-2 with count 3
 // 2-2.5 with count 1
+
```

Comment:
also add a test around the maxArrayValue boundary, I want to confirm there isn't an off by 1 error 

so test that if you record 999 it ends up in the >= 900 < 1000 bucket
same for 999.99

check that 1000 shows up in the >= 1000 < 2000 bucket
same for 1000.01 and 1001 and 1999.99

check that 2000 and 2001 show up in >= 2000 bucket (the first virtual bucket)

ideally should be table driven like the round test and not like the early tests checking the text output (which is somewhat brittle)